.github/ISSUE_TEMPLATE/releases/release_checklist.md

```diff
@@ -56,7 +56,17 @@
            - [ ] Approve the `Approval - Partner notification` approval stage.
       1. - [ ] `Approval - Release` stage
            - [ ] **SYNC POINT**: Wait for Microsoft build release.
-           - [ ] ⚠️ 8.0: Verify the release tag in the `dnceng/dotnet-security-partners` repository matches theVMR commit to which is shown in the `Pre-Release` stage (in the `Get Associated Pipeline Run IDs` step).
+           - [ ] ⚠️ 8.0: Verify the release tag in [dotnet-security-partners](https://dev.azure.com/dotnet-security-partners/_git/dotnet) repository matches the VMR commit to which is shown in the `Pre-Release` stage (in the `Get Associated Pipeline Run IDs` step).
+           - [ ] ⚠️ If a custom tag was needed for this servicing release, mirror it to public repo.
```

Comment:
Is there a tracking issue to automate this?  If not please log one.  Either way the tracking issue should be referenced here.

Reply:
There is an issue for adding manual steps to checklist https://github.com/dotnet/source-build/issues/3585

I will create an issue for automating those steps and adding to release pipeline. I'm not sure of the location for this issue reference - should it be in checklist document?

Reply:
Created an issue: https://github.com/dotnet/source-build/issues/3594

Reply:
> Either way the tracking issue should be referenced here.

My 2 cents: I don't like referencing issues from the release checklist because each release will end up referencing that issue. So the referenced issue will have a bunch of back-references to it, none of which are really relevant.